sensor_msgs/PointCloud2 point_cloud
---
iri_tos_supervoxels/segmented_objects objects
